provision/acc_provision/templates/aci-installer-cr.yaml

```diff
@@ -5,9 +5,13 @@ metadata:
   name: installer-aci-cni
 spec:
   config:
+    connectivityCheck:
```

Comment:
This entire "connectivityCheck" section should only exist if config.connectivity_checker field exists in the input file. Since this is optional to set by the user. No "else" required

Reply:
addressed